ssl/s3_enc.c

```diff
@@ -268,7 +268,7 @@ int ssl3_setup_key_block(SSL *s)
     SSL_COMP *comp;
 
     if (s->s3->tmp.key_block_length != 0)
-        return (1);
+        return (0);
```

Comment:
It isn't clear to me under what circumstances this check would fail and whether it is an error (as you have made it), or should be treated as success (as it was). What is your justification for this change?